cli/lib/errors.js

```diff
@@ -94,6 +95,35 @@ const unexpected = {
   `,
 }
 
+const removed = {
+  CYPRESS_BINARY_VERSION: {
+    description: stripIndent`
+    The environment variable CYPRESS_BINARY_VERSION has been removed as of version ${chalk.green('3.0.0')}
+    `,
+    solution: stripIndent`
+      You should use the equivalent environment variable CYPRESS_INSTALL_BINARY instead.
+    `,
+  },
+  CYPRESS_SKIP_BINARY_INSTALL: {
+    description: stripIndent`
+    The environment variable CYPRESS_SKIP_BINARY_INSTALL has been removed as of version ${chalk.green('3.0.0')}
+    `,
+    solution: stripIndent`
+      To skip the binary install, set CYPRESS_INSTALL_BINARY=0
```

Comment:
I find this option of setting `0` to be a bit weird. Don't have an alternative suggestion, just feels weird.

Reply:
well, we could keep CYPRESS_SKIP_BINARY_INSTALL